src/gherkin/stepDef.test.js

```diff
@@ -138,7 +138,7 @@ describe('Checking stepDef generated patterns', () => {
         const stepDef = new StepDef(data.stepDef.token, data.stepDef.pattern, () => {
           return 0
         })
-        throw new Error('creating invalid stepdef should fail but it passed', stepDef)
+        throw new Error('creating invalid stepdef should fail but it passed')
```

Comment:
`Error` only takes one parameter. 

If I put `+ stepDef` then maybe something will go wrong, or some really long sting representation of `stepDef` will be output.